plugins/modules/pnp_intent.py

```diff
@@ -597,12 +614,13 @@ def get_have(self):
 
                 if site_exists:
                     have["site_id"] = site_id
-                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id))
-                    self.log("Site Name:" + str(site_name))
+                    self.log("Site Exists: " + str(site_exists) + "\n Site_id:" + str(site_id), "INFO")
+                    self.log("Site Name:" + str(site_name), "INFO")
                     if self.want.get("pnp_type") == "access_point":
                         if self.get_site_type() != "floor":
                             self.msg = "Type of the site must \
```

Comment:
self.msg = "The site type must be specified as 'floor' for claiming an AP."

Reply:
Makes sense